lms/events/event.py

```diff
@@ -66,12 +66,7 @@ def _get_role_ids(self):
         return [role.id for role in self.request.lti_user.lti_roles]
 
     def _get_application_instance_id(self):
-        if application_instance := self.request.find_service(
```

Comment:
This code was wrong, get_current never returns None, it raises.

But also, LTIEvent implies "LTIUser" and a AI attached to it so we never seen it fail before.

Removed the test case for this below.